src/core/symbology/qgsinterpolatedlinerenderer.cpp

```diff
@@ -903,6 +903,11 @@ QgsSymbolLayer *QgsInterpolatedLineSymbolLayer::create( const QVariantMap &prope
   return symbolLayer.release();
 }
 
+Qgis::SymbolLayerFlags QgsInterpolatedLineSymbolLayer::flags() const
+{
+  return Qgis::SymbolLayerFlag::DisableFeatureClipping;
```

Comment:
@nyalldawson , nice idea -- it'd be useful/needed for the arrow symbol layer too.